diff --git a/dist/telemetry-system-Pyz9irBj.mjs b/dist/telemetry-system-Pyz9irBj.mjs
--- a/dist/telemetry-system-Pyz9irBj.mjs
+++ b/dist/telemetry-system-Pyz9irBj.mjs
@@ -1,3 +1,3 @@
 import { SeverityNumber as SeverityNumber$1 } from "@opentelemetry/api-logs";
-import { Resource } from "@opentelemetry/resources";
+import { resourceFromAttributes } from "@opentelemetry/resources";
 import { ATTR_SERVICE_NAME } from "@opentelemetry/semantic-conventions";
@@ -1004 +1004 @@
-	const resource = new Resource(resourceAttributes);
+	const resource = resourceFromAttributes(resourceAttributes);
@@ -1046,5 +1046,8 @@
-	loggerProvider = new LoggerProvider({ resource });
-	loggerProvider.addLogRecordProcessor(new BatchLogRecordProcessor(logExporter, {
-		scheduledDelayMillis: logsScheduledDelayMillis,
-		maxExportBatchSize: logsMaxExportBatchSize
-	}));
+	loggerProvider = new LoggerProvider({
+		resource,
+		processors: [new BatchLogRecordProcessor({
+			exporter: logExporter,
+			scheduledDelayMillis: logsScheduledDelayMillis,
+			maxExportBatchSize: logsMaxExportBatchSize
+		})]
+	});
diff --git a/dist/telemetry-system-BH119njX.cjs b/dist/telemetry-system-BH119njX.cjs
--- a/dist/telemetry-system-BH119njX.cjs
+++ b/dist/telemetry-system-BH119njX.cjs
@@ -1004 +1004 @@
-	const resource = new _opentelemetry_resources.Resource(resourceAttributes);
+	const resource = (0, _opentelemetry_resources.resourceFromAttributes)(resourceAttributes);
@@ -1046,5 +1046,8 @@
-	loggerProvider = new _opentelemetry_sdk_logs.LoggerProvider({ resource });
-	loggerProvider.addLogRecordProcessor(new _opentelemetry_sdk_logs.BatchLogRecordProcessor(logExporter, {
-		scheduledDelayMillis: logsScheduledDelayMillis,
-		maxExportBatchSize: logsMaxExportBatchSize
-	}));
+	loggerProvider = new _opentelemetry_sdk_logs.LoggerProvider({
+		resource,
+		processors: [new _opentelemetry_sdk_logs.BatchLogRecordProcessor({
+			exporter: logExporter,
+			scheduledDelayMillis: logsScheduledDelayMillis,
+			maxExportBatchSize: logsMaxExportBatchSize
+		})]
+	});
